### PR TITLE
Add the ctitle command

### LINKED ISSUE #282: Suggestion: Add the `ctitle` command
Currently, there is no way (that I know of) to display title text (in the title, subtitle, or action bar slots) completely client side. Other mods do display title text client side, which shows that it can be done. Such title text could be used to mark milestones (such as "Congratulations!" "You reached day 100!") or other events. To add to this, a `ctitle` command could be added to the mod, which works exactly the same as `/title`, but has no `<targets>` argument (which always defaults to yourself).

### PR DESCRIPTION
**Pull request by haykam821**
Fixes #282

### CODE REVIEW

**Comment by xpple** [comment]
Please note that this was not intended to be in clientcommands, but rather intended as a script. I’m referring to [this message](https://discord.com/channels/703363189081112647/703598704266969099/865709229137199115).

**Comment by Earthcomputer** [comment]
Eh whatever, it's not much work for me

File: src/main/java/net/earthcomputer/clientcommands/command/CTitleCommand.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import net.minecraft.text.TranslatableText;
+
+import static com.mojang.brigadier.arguments.IntegerArgumentType.*;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.command.argument.TextArgumentType.*;
+import static net.minecraft.server.command.CommandManager.*;
+
+public class CTitleCommand {
+
+    private static final MinecraftClient CLIENT = MinecraftClient.getInstance();
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/CTitleCommand.java]: The `MinecraftClient` is not immutable, all caps is for immutable objects

File: src/main/java/net/earthcomputer/clientcommands/command/CTitleCommand.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    .then(argument("stay", integer(0))
+                        .then(argument("fadeOut", integer(0))
+                            .executes(ctx -> executeTimes(getInteger(ctx, "fadeIn"), getInteger(ctx, "stay"), getInteger(ctx, "fadeOut"))))))));
+    }
+
+    private static int executeClear() {
+        CLIENT.inGameHud.clearTitle();
+
+        sendFeedback(new TranslatableText("commands.ctitle.cleared"));
+        return 1;
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/CTitleCommand.java]: return 0

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

**Comment by Earthcomputer** [comment]
This PR needs updating after the fcapi merge

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**